docs/how-meeting-recordings-upload-works.md

```diff
@@ -0,0 +1,29 @@
+# How automatic video upload works
+
+Videos are being recorded using one of [zoom accounts](https://github.com/open-telemetry/community/blob/main/assets.md#zoom-accounts) used by OpenTelemetry. OpenTelemetry zoom accounts are paid for and owned by CNCF.
+
+## Video upload process
```

Comment:
Maybe include a point about how long it takes for a video to become public. I have seen instance where this can take quite a few days. We don't want users creating issues for videos that are only a couple hours old.

Reply:
added a note